conf/logback.xml

```diff
@@ -35,17 +35,18 @@
     </appender>
 
 
-    <logger name="accesslog" level="INFO" additivity="false">
-        <appender-ref ref="ACCESS_LOG_FILE" />
-    </logger>
-
-    <logger name="com.ning.http.client.providers.netty" additivity="false">
-        <appender-ref ref="STDOUT_IGNORE_NETTY" />
-    </logger>
-
-    <logger name="com.google.inject" level="INFO"/>
-    <logger name="org.asynchttpclient.netty.channel.DefaultChannelPool" level="INFO"/>
-    <logger name="io.netty" level="INFO"/>
+<!--    <logger name="accesslog" level="INFO" additivity="false">-->
```

Comment:
Do we need this?

Reply:
oops... no I'll add back in. Was messing with the logs to try and see why something wasn't happening.

Reply:
will fix up